### PR TITLE
Feature/gsap jo v2

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description drawing

- Add gsap logic to animate the `pokemonDetails ``Img`
- Add "Easter egg" 
- Add gsap logic to animate the pokeball

Next steps:

Still need to finish the animations ( decide how we want it to be- I would love some help here)
Deal with merging conflicts ( I had to manipulate the Components App and header )

*** *I saw a merging conflict that doesn't make sense to me in this branch... 
I've tried to fix alone, but it showed me a button to `Commit Merge` to be able to fix it.
I wasn't sure if that would merge the code at once so I've decided to keep it like that and talk to you before doing it!*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [ ] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,8 +1,31 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import './Header.css'
 import { Route, Link } from 'react-router-dom';
+import { gsap } from 'gsap';
 
-const Header = ( clearPokemon ) => {
+const Header = ({ foundPokemon }) => {
```
Comment (hayleyw7) [inline src/components/Header/Header.js]: Great way to add this in at the right time.

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -24,7 +24,6 @@
 .how-to {
   font-size: 50px;
   color: white;
-  /* color:#f2b32e; */
```
Comment (hayleyw7) [inline src/components/Header/Header.css:27]: Thanks for making leeway on refactoring as we go.

File: src/components/PokemonDetails/PokemonDetails.js [hunk 3]
```diff
@@ -68,36 +47,41 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage, clearPokemon }) => {
...
+  },[pokemonDetails]);
  
-
   if (pokemonDetails.types) {
     return (
       <div className="pokemon-details-page">
         <div className="pokemon-details-container">
-
-
+          <audio className="pokemon-audio"></audio>
```
Comment (hayleyw7) [inline src/components/PokemonDetails/PokemonDetails.js:81]: This cracks me up. Fun Easter Egg.

File: src/components/PokemonDetails/PokemonDetails.js [hunk 4]
```diff
@@ -68,36 +47,41 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage, clearPokemon }) => {
...
-  // //   {
-  // //     opacity: 1,
-  // //     x: 10,
-  // //     y: 10,
-  // //     ease: Power3.easeOut
-  // //   }
-  // // )
-
-  // },[])
+  useEffect(() => {
```
Comment (hayleyw7) [inline src/components/PokemonDetails/PokemonDetails.js:50]: I love this animation.

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Header/Header.js [hunk 5]
```diff
@@ -1,8 +1,31 @@
...
+      rotation: 720
+    },
+    {   
+        scale: 1,
+        opacity: 1,
+        duration: 2.5, 
+        ease: "rough({ template: none.out, strength: 1, points: 20, taper: 'none', randomize: true, clamp: false})", 
+        y: 0,
+       
+        rotation: 0
```
Comment (Matt-Roden) [inline src/components/Header/Header.js:24]: This is such a cool effect! I like how you made the ball look like it was in reverse

File: src/components/PokemonDetails/PokemonDetails.js [hunk 6]
```diff
@@ -68,36 +47,41 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage, clearPokemon }) => {
...
+            duration: 2.5, 
+            ease: "rough({ template: none.out, strength: 1, points: 20, taper: 'none', randomize: true, clamp: false})", 
+            y: 0,
+            scale: 1,
+            rotation: 0
+        })
+      if (pokemonDetails.name === "pikachu") {
+            var audio  = document.getElementsByClassName("pokemon-audio")[0]
+            audio.loop = false;
+            audio.src  = `https://play.pokemonshowdown.com/audio/cries/${pokemonDetails.name}.ogg`;
```
Comment (Matt-Roden) [inline src/components/PokemonDetails/PokemonDetails.js:72]: Pika!

**Review by Matt-Roden — APPROVED** [review]
Good job, these animations are awesome!

**Pull request merged**